docs/projects/fps/tutorial.md

```diff
@@ -113,13 +113,13 @@ The `HealthPickup` entity is a new type of entity, so we must create a new entit
 **Step 3.** Define a new static function within the `FpsEntityTemplates` class which takes the position of the health pack and the value of the health pack as parameters and returns an `EntityTemplate` instance:
 
 ```csharp
-public static EntityTemplate HealthPickup(Vector3f position, uint healthValue)
+public static EntityTemplate HealthPickup(Vector3 position, uint healthValue)
 {
     // Create a HealthPickup component snapshot which is initially active and grants "heathValue" on pickup.
     var healthPickupComponent = new Pickups.HealthPickup.Snapshot(true, healthValue);
 
     var entityTemplate = new EntityTemplate();
-    entityTemplate.AddComponent(new Position.Snapshot(new Coordinates(position.X, position.Y, position.Z)), WorkerUtils.UnityGameLogic);
+    entityTemplate.AddComponent(new Position.Snapshot(Coordinates.FromUnityVector(position), WorkerUtils.UnityGameLogic);
```

Comment:
does position.ToCoordinates() not work?

Reply:
Unsure which imports are required for that, so I went with this 😬